app/src/test/java/waiter/RouterTest.java

```diff
@@ -68,14 +67,14 @@ void returnsNotFoundWithReasonResponseForExistingUrlButNoGetMethodInRoute() {
         Response Response = new Router(routes).getRequestedResponse(request);
 
         assertEquals("HTTP/1.1", Response.getProtocol());
-        assertEquals(waiter.Response.Status.NotFound, Response.getStatus());
-        assertEquals("Content-Length: " + Response.getBody().length(), Response.getHeaders());
-        assertEquals("404, Found resource but no corresponding method", Response.getBody());
+        assertEquals(waiter.Response.Status.MethodNotAllowed, Response.getStatus());
+        assertEquals("Content-Length: 0", waiter.Response.HeaderField.ContentLength.asString + Response.getHeaders().get(waiter.Response.HeaderField.ContentLength));
```

Comment:
Have to break this up into something more readable.

Reply:
Will create an intermediary variable for readability 👍 thank you!

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/20/commits/341ff7eb3f6f37811672f55c1b3d1248781533da), it may look a wee bit wonky as I made a git mistake, just wanted to mention